frontends/web/package.json

```diff
@@ -38,7 +38,7 @@
     "react-i18next": "14.1.2",
     "react-router-dom": "6.4.3",
     "react-select": "5.8.0",
-    "request-address": "0.0.10"
+    "request-address": "pocketbitcoin/request-address#payment"
```

Comment:
@TKone7 please let me know when 0.0.11 is published on npm https://www.npmjs.com/package/request-address